### DISCUSSION TITLE
Custom User Profile Property is not being crawled for all the users in SharePoint Online. Only a few updates are reflected in pnp search even after waiting for 24 hours.

### DISCUSSION THREAD

**Discussion by GovindSurnidi** [microsoft-search/pnp-modern-search#3176] [General]
Custom User Profile Property is not being crawled for all the users in SharePoint Online. Only a few updates are reflected in pnp search even after waiting for 24 hours.

I have SharePoint administrator access on the tenant and have updated a custom User Profile property for about 150 users with different values. 
That custom user property was already mapped to a refinablestring in managed property in search schema. 

The mapped refinablestring is selected in the properties for pnp search results webpart and to pnp search filters webpart. 
After waiting for about 24 hours, I have noticed that in the pnp search filters dropdown only a few values from refinablestring field are available, but some other values are missing. I have waited for another 24hours hoping all the values will be crawled and be available in the filter's dropdown, but no luck.

Any suggestions/recommendations in fixing this issue are highly appreciated. 
Thanks

**Comment by kasperbolarsen** [comment]
Reindexing the User Profiles using this script usually solve that kind of issues: https://adoption.microsoft.com/en-us/sample-solution-gallery/sample/spo-request-pnp-reindex-user-profile/

Ps, this issue has nothing to do with PnP Modern Search web parts 

I'll recommend you use a tool such as this when checking managed properties as it query the API directly: https://github.com/SharePoint/PnP-Tools/blob/master/Solutions/SharePoint.Search.QueryTool/README.md
